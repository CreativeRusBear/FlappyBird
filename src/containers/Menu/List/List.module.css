.List {
  display: flex;
  flex-flow: column;
  gap: 14px;
}

.List button {
    background-color: var(--yellow-color);
    border-radius: 4px;
    width: 240px;
    text-align: center;
    border: none;
    padding: 0;
}



.List button a {
    color: var(--white-color);
    font-size: 18px;
    padding: 12px 0;
    width: 100%;
}
