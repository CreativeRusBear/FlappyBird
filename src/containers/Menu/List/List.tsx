import React from 'react';
import {NavLink} from 'react-router-dom';
import classes from './List.module.css';

const List = () => (
  <div className={classes.List}>
    <button>
      <NavLink to='/game'>Play</NavLink>
    </button>
    <button>
      <NavLink to='/author'>Author</NavLink>
    </button>
  </div>
);

export default List;
