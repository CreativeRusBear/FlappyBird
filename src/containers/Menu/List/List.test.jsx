import React from "react";
import List from "./List";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";

describe('<List />', () => {
  test('should renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <List />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
