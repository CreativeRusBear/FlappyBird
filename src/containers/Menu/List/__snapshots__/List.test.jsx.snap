// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should renders correctly 1`] = `
<div
  className="List"
>
  <button>
    <a
      aria-current={null}
      href="/game"
      onClick={[Function]}
    >
      Play
    </a>
  </button>
  <button>
    <a
      aria-current={null}
      href="/author"
      onClick={[Function]}
    >
      Author
    </a>
  </button>
</div>
`;
